game.js

```diff
@@ -27,52 +27,85 @@ function getRandom(array) {
   return array[idx];
 }
 
-/**
- * Adds and removes randomly selected classes to randomly selected trumps
- *
- */
-function randTrump() {
-  // Available classes for trump position
-  const positions = [
+// Available classes for trump position
+const positions = [
     'animateTop',
     'animateLeft',
     'animateRight'
-  ];
+];
 
-  // Available classes for trumps
-  const trumps = [
+// Available classes for trumps
+const trumps = [
     '.trump01',
     '.trump02',
     '.trump03',
     '.trump04',
     '.trump05',
     '.trump06'
-  ];
-  setInterval(() => {
-    let randomTrump = document.querySelector(
-      getRandom(trumps)
-    );
+];
+
+// Starts random trump appearance
+let gameActive = setInterval(() => {
+    let randomTrump = document.querySelector(getRandom(trumps))
     let randPosition = getRandom(positions);
     randomTrump.classList.add('animate', randPosition);
     setTimeout(() => {
-      randomTrump.classList.remove('animate', randPosition);
-    }, 1500);
-    console.log(randomTrump);
-  }, randTime(2000, 4000));
-}
-
-randTrump();
+            randomTrump.classList.remove('animate', randPosition);
+        },1500)
+}, randTime(2000, 4000));
 
 let trumpsDiv = document.querySelectorAll('.trump');
 let score = 0;
 
-//Wrap in if statement to check if trump is popped up first
-trumpsDiv.forEach((trump) =>
+// Animates on click and adds to score
+trumpsDiv.forEach(trump =>
   trump.addEventListener('click', () => {
     if (trump.classList.contains('animate')) {
       trump.classList.remove('animate');
       score += 1;
       document.querySelector('.score').textContent = score;
     }
-  })
-);
+}))
+
+const modalContainer = document.querySelector(".modalContainer")
+const modal = document.querySelector(".modal")
```

Comment:
Doesnt like like this constant ever gets used

Reply:
Yep, meant to remove that one too!